milabench/metrics/sqlalchemy.py

```diff
@@ -0,0 +1,350 @@
+from collections import defaultdict
+from datetime import datetime
+from dataclasses import dataclass
+
+from ..structs import BenchLogEntry
+
+from bson.json_util import dumps as to_json
+from bson.json_util import loads as from_json
+import sqlalchemy
+from sqlalchemy.dialects import postgresql
```

Comment:
Yes, the line `from sqlalchemy.dialects import postgresql` is use uniquely to generate the table creation script that is necessary for PostgreSQL since I assume that the milabench user does not have table creation permission on the database.

For SQLite, the tables are created if the database does not exists.

Tests run for both SQLite and PostgreSQL.